src/helpers/productInformation.js

```diff
@@ -0,0 +1,24 @@
+import {bestSellingTv} from '../constants/inventory.js';
+
+function productInformation() {
+    return (bestSellingTv.brand + ' ' + bestSellingTv.type + ' - ' + bestSellingTv.name);
```

Comment:
Dit zou je ook korter kunnen schrijven op de volgende manier:
`${tv.brand} ${tv.type} - ${tv.name}`;

Maar beide is helemaal prima!

Reply:
Thanks! Had ik ik inmiddels overal aangepast toen ik eenmaal weer wist hoe het geschreven moest worden ;)